index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <div>
+      <nav class="navbar">
+        <img src="/assets/My Logo.svg" class="logo" />
+        <img class="menu" src="assets/Side.svg" alt="menu" />
+      </nav>
+      <section class="my-section">
+        <div>
+          <h1 class="intro">I'm Walid <br />Glad to see <br />You!</h1>
+        </div>
+        <div>
+          <p class="about-me">
+            I’m a software developer! I can help you <br />
+            build a product , feature or website Look <br />
+            through some of my work and <br />
+            experience! If you like what you see and <br />
+            have a project you need coded, don’t <br />
+            hestiate to contact me.
+          </p>
+        </div>
+        <div class="contact-me">
+          <h2 class="connect">LET'S CONNECT</h2>
+        </div>
+        <div class="buttons">
+          <a href=""><img src="/assets/Github.svg" alt="Github" /></a>
```

Comment:
- [ ] Please Headline section should  has the margins (top, bottom, left, right) defined in the design

| Your code  | Figma code |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/211890782-2598ac66-43ef-42da-ab6f-cda252c10a39.png)  | ![image](https://user-images.githubusercontent.com/74744787/211890999-3269efaa-b77c-4119-af6f-4b8255dbdc07.png)  |